### PR TITLE
Seed verify mobile.

### LINKED ISSUE #582: Implement Seed Verify mobile screen
https://invis.io/SENS97E65WH

### PR DESCRIPTION
**Pull request by valentinewallace**
Closes #582 and #581.

### CODE REVIEW

**Comment by tanx** [comment]
@valentinewallace also here please link the issue ... it's easier to follow the links when reviewing. Thanks :)

File: src/action/auth-mobile.js [hunk 1]
```diff
@@ -135,6 +135,55 @@ class AuthAction {
...
+    if (
+      seedVerify[0] !== seedMnemonic[seedVerifyIndexes[0] - 1] ||
+      seedVerify[1] !== seedMnemonic[seedVerifyIndexes[1] - 1] ||
+      seedVerify[2] !== seedMnemonic[seedVerifyIndexes[2] - 1]
+    ) {
+      this._alert("Seed words don't match", () => {});
+      return;
+    }
+    this.initSetPin();
+  }
```
Comment (tanx) [inline src/action/auth-mobile.js]: The auth controller is only for pin entry and fingerprint. Seed stuff should go in a different controller. Any reason we can't reuse the logic from the desktop?

Comment (valentinewallace) [reply src/action/auth-mobile.js]: Fixed!

Comment (valentinewallace) [reply src/action/auth-mobile.js]: My mistake, fixed! Thought that only referred to `checkSeed` for some reason.

**Comment by tanx** [comment]
Closing in favor of #878

**Pull request closed without merging**